pkg/api/features.go

```diff
@@ -37,6 +37,6 @@ func (a *BasicFeatureAttributes) Description() string {
 	return a.FeatureDescription
 }
 
-func FeatureSorter[A FeatureAttributes, F Feature[A]](a F, b F) int {
+func FeatureSorter[A FeatureAttributes, B any, F Feature[A, B]](a F, b F) int {
```

Comment:
We should probably create a specific type or interface for this to have a narrower scope